user/downloading-installing-upgrading/supported-versions.md

```diff
@@ -127,6 +127,15 @@ To be precise:
 
 We aim to announce both types of events one month in advance in order to remind users to upgrade.
 
+### Note on custom TemplateVMs
+
+[QubesBuilder](/doc/qubes-builder/) includes both experimental TemplateVMs (ex: fc31 backported to R4.0) and community-contributed builders for custom TemplateVMs (ex: kali). They are not supported by QubesOS itself and not included in QubesOS ISOs, but can be built by experienced users.
+
+| Qubes OS      | TemplateVM    | Status       |
+| ------------- | ------------- | ------------ |
+| Release 4.0   | Fedora 31 | Fail |
+| Release 4.0   | Kali      | Fail |
+| Release 4.1   | Kali      | Builds |
```

Comment:
Is that really true? AFAIR the problem was missing `salt` package in Debian bullseye, which shouldn't depend on Qubes version. See https://tracker.debian.org/pkg/salt

Reply:
No idea, I didn't investigate the fail. I just reported what I've got. Feel free to adapt the table itself.

Reply:
Kali template fails to build regardless of qubes version.